fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -2810,7 +2810,7 @@ public void sendStmtPrepareOK(int stmtId, List<String> labels) throws IOExceptio
             context.getMysqlChannel().sendOnePacket(serializer.toByteBuffer());
         }
         context.getMysqlChannel().flush();
-        context.getState().setNoop();
```

Comment:
what happend if setOK? you should distingwish diffent behavior bettwen NOOP and OK

Reply:
After verification, I found that changing it to OK would cause issues. Therefore, when logging, I added a validation: if the status is `com_stmt_prepare` and there are no errors, it is recorded as OK.

Reply:
I saw the issue and this setting, and I thought this setNoop was a bug, so I didn’t think it through carefully. Thank you for the reminder